portal/migrations/0002_fix_ip_type.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 5.1.7 on 2025-03-23 22:38
+
+from django.db import migrations
+from django.db.utils import OperationalError
+from django.conf import settings
+
+
+def forwards(apps, schema_editor):
+    if not schema_editor.connection.alias == 'default':
+        return
+    # postgresql only
+    try:
+        schema_editor.execute("ALTER TABLE portal_ipaddress ALTER COLUMN ip TYPE inet USING(ipinet);")
+    except OperationalError:
+        import sys
+        sys.stderr.write("Warning: could not set TYPE inet on ip field. Only supported on postgres.\n")
```

Comment:
not sure if this matters, but if this can safely be ignored on other db engines, it might be nice to say so.